lte/gateway/c/oai/tasks/mme_app/mme_app_bearer.c

```diff
@@ -84,6 +84,10 @@
 extern task_zmq_ctx_t mme_app_task_zmq_ctx;
 extern int pdn_connectivity_delete(emm_context_t* emm_context, pdn_cid_t pid);
 
+static void send_s11_modify_bearer_request(
```

Comment:
if this is intended for S8 task only then I suggest to use something like `send_modify_bearer_request_to_S8task`.

Reply:
never mind I have missed that this is splitting the request among S8 and and SPGW tasks.

Reply:
No problem